### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'by'

### ISSUE THREAD

**Issue by prgarnett** [GjjvdBurg/signal2html#57]
2021-12-16 15:14:06 | INFO - This is signal2html version 0.2.7
2021-12-16 15:14:06 | INFO - Using database version 120.
2021-12-16 15:14:06 | WARNING - Found untested Signal database version: 120.

I think the error:
AttributeError: 'NoneType' object has no attribute 'by'

Might be caused by group chats perhaps?

**Comment by GjjvdBurg** [comment]
Hi @prgarnett, thanks for reporting this issue. Can you post the full output that you see?

**Comment by aweinberg38** [comment]
I have the same issue as prgarnett 
Full output below
-----
signal2html -i signal_db -o signal_output
2021-12-19 23:08:05 | INFO - This is signal2html version 0.2.7
2021-12-19 23:08:05 | INFO - Using database version 120.
2021-12-19 23:08:05 | WARNING - Found untested Signal database version: 120.
Traceback (most recent call last):
  File "/usr/local/bin/signal2html", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.9/dist-packages/signal2html/__main__.py", line 20, in main
    sys.exit(main())
  File "/usr/local/lib/python3.9/dist-packages/signal2html/ui.py", line 35, in main
    process_backup(args.input_dir, args.output_dir)
  File "/usr/local/lib/python3.9/dist-packages/signal2html/core.py", line 677, in process_backup
    populate_thread(
  File "/usr/local/lib/python3.9/dist-packages/signal2html/core.py", line 634, in populate_thread
    sms_records = get_sms_records(db, thread, addressbook)
  File "/usr/local/lib/python3.9/dist-packages/signal2html/core.py", line 99, in get_sms_records
    data = get_data_from_body(_type, body, addressbook, _id)
  File "/usr/local/lib/python3.9/dist-packages/signal2html/core.py", line 377, in get_data_from_body
    data = get_group_update_data_v2(
  File "/usr/local/lib/python3.9/dist-packages/signal2html/core.py", line 312, in get_group_update_data_v2
    if change.by:
AttributeError: 'NoneType' object has no attribute 'by'

**Comment by prgarnett** [comment]
Thanks for posting that as I am away from my computer for a while.

**Comment by GjjvdBurg** [comment]
Thanks for posting the error message @aweinberg38! I think the fix in #58 should do it. If either of you wants to test this, you can use the following commands:

```
$ pip uninstall signal2html
$ pip install -U git+https://github.com/GjjvdBurg/signal2html.git@bugfix/group_update_v2
```

**Comment by GjjvdBurg** [comment]
Now implemented in version 0.2.8.
